doc/nrf/installation/install_ncs.rst

```diff
@@ -0,0 +1,617 @@
+
+         .. west-error-start
+
+         .. note::
+
+            If you get an error message when running west, :ref:`update west <west_update>` to the latest version.
+            See :ref:`zephyr:west-troubleshooting` in the Zephyr documentation for more information.
+
+         .. west-error-end
+
+      #. Enter the following command to clone the project repositories:
+
+         .. code-block:: console
+
+            west update
+
+         Depending on your connection, this might take some time.
+      #. Export a :ref:`Zephyr CMake package <zephyr:cmake_pkg>`.
+         This allows CMake to automatically load the boilerplate code required for building |NCS| applications:
+
+         .. code-block:: console
+
+            west zephyr-export
+..
+
+If you used the default locations, your directory structure now looks similar to this:
+
+.. code-block:: none
+
+   <home>/
+   ├─── toolchains/
+   │  └─── <toolchain-installation>
+   └─── <west-workspace>/
+      ├─── .west/
+      ├─── bootloader/
+      ├─── modules/
+      ├─── nrf/
+      ├─── nrfxlib/
+      ├─── zephyr/
+      └─── ...
+
+In this simplified structure preview, *<home>* corresponds to :file:`ncs/` and *<toolchain-installation>* and *<west-workspace>* correspond to the version names you installed.
+There are also additional directories, and the structure might change over time, for example if you later :ref:`change the state of development to a different revision <updating_repos>`.
+The full set of repositories and directories is defined in the :ref:`manifest file <zephyr:west-manifest-files>` (`see the file in the repository <west manifest file_>`_).
+
+.. _build_environment_cli:
+
+Set up the command-line build environment
+*****************************************
+
+In addition to the steps mentioned above, if you want to build and program your application from the command line, you have to set up your command-line build environment by defining the required environment variables every time you open a new command-line or terminal window.
+See :ref:`zephyr:important-build-vars` for more information about the various relevant environment variables.
+
+Define the required environment variables as follows, depending on your operating system:
+
+.. tabs::
+
+   .. tab:: Windows
+
+      Navigate to the :file:`ncs` directory and run the following command in a terminal window:
+
+      .. code-block:: console
+
+         zephyr\zephyr-env.cmd
+
+      If you need to define additional environment variables, create the file :file:`%userprofile%\zephyrrc.cmd` and add the variables there.
+      This file is loaded automatically when you run the above command.
+      See :ref:`Zephyr documentation about using zephyrrc files <zephyr:env_vars_zephyrrc>` for more information.
+
+   .. tab:: Linux
+
+      Navigate to the :file:`ncs` directory and run the following command in a terminal window:
+
+      .. code-block:: console
+
+         zephyr\zephyr-env.cmd
```

Comment:
incorrect

Reply:
Let me hide out of shame for the copy-paste error. Thanks for spotting!